mrblib/mitamae/inline_backends/file_backend.rb

```diff
@@ -19,6 +19,22 @@ def get_file_mode(path)
       rescue => e
         Specinfra::CommandResult.new(stdout: '', stderr: e.message, exit_status: 1)
       end
+
+      def get_file_owner_user(path)
+        uid = File.stat(path).uid
+        passwd = Etc.getpwuid(uid)
+        Specinfra::CommandResult.new(stdout: passwd.name, stderr: '', exit_status: 0)
+      rescue => e
+        Specinfra::CommandResult.new(stdout: '', stderr: e.message, exit_status: 1)
```

Comment:
+1 for adding a different class which acts like Specinfra::CommandResult.

Previously, `run_command` case raises an error when the command fails, which is different from this patch.
I think raising the original exception is better than logging exceptions and it's more similar to the previous behavior.
I'm preparing a patch for it.